### PR TITLE
issue-668: Add test case for LogUtils.

### PR DESCRIPTION
**Pull request by HoldDie**
// Describe your PR here; eg. Fixes #issueNo

Make sure that:

- [x] You have read the [contribution guidelines](https://dromara.org/en-us/docs/soul/contributor.html).
- [x] You submit test cases (unit or integration tests) that back your changes.
- [x] Your local test passed `mvn clean install -Dmaven.javadoc.skip=true`.

### CODE REVIEW

File: soul-common/src/test/java/org/dromara/soul/common/utils/LogUtilsTest.java [hunk 1]
```diff
@@ -0,0 +1,90 @@
...
+import org.slf4j.LoggerFactory;
+
+import java.util.function.Supplier;
+
+/**
+ * Test case for {@link LogUtils}.
+ *
+ * @author HoldDie
+ */
+public class LogUtilsTest {
```
Comment (yu199195) [inline soul-common/src/test/java/org/dromara/soul/common/utils/LogUtilsTest.java:33]: add final

File: soul-common/src/test/java/org/dromara/soul/common/utils/LogUtilsTest.java [hunk 2]
```diff
@@ -0,0 +1,90 @@
...
+
+    @Test
+    public void testGetInstance() {
+        final LogUtils result = LogUtils.getInstance();
+        Assert.assertNotNull(result);
+    }
+
+    @Test
+    public void testDebugWithFormat() {
+        LogUtils.debug(logger, "testDebug: {}", supplier);
```
Comment (yu199195) [inline soul-common/src/test/java/org/dromara/soul/common/utils/LogUtilsTest.java:53]: can you assert debug info Is it what you expected？

File: soul-common/src/test/java/org/dromara/soul/common/utils/LogUtilsTest.java (hunk 1 above)
Comment (HoldDie) [reply soul-common/src/test/java/org/dromara/soul/common/utils/LogUtilsTest.java:33]: ok

File: soul-common/src/test/java/org/dromara/soul/common/utils/LogUtilsTest.java (hunk 2 above)
Comment (HoldDie) [reply soul-common/src/test/java/org/dromara/soul/common/utils/LogUtilsTest.java:53]: yes

**Review by yu199195 — APPROVED** [review]

**Pull request merged**